webui/react/src/components/DatePickerFilter.module.scss

```diff
@@ -0,0 +1,5 @@
+.wrapper {
+  :global(.ant-picker) {
+    border-radius: var(--theme-sizes-border-radius);
+  }
```

Comment:
**suggestion:** This looks useful to have across `Antd` so place `.ant-picker` in `antd.scss`